internal/pkg/build/conveyorPacker.go

```diff
@@ -30,8 +30,8 @@ type ConveyorPacker interface {
 	Packer
 }
 
-// conveyorPacker returns a valid ConveyorPacker for the given image definition.
-func conveyorPacker(def types.Definition) (ConveyorPacker, error) {
+// NewConveyorPacker returns a valid ConveyorPacker for the given image definition.
+func NewConveyorPacker(def types.Definition) (ConveyorPacker, error) {
 	switch def.Header["bootstrap"] {
```

Comment:
Same here, might be best to return an `error` if `def.Header["bootstrap"]` isn't defined?